core/dbt/adapters/base/impl.py

```diff
@@ -92,6 +95,14 @@ def _expect_row_value(key: str, row: agate.Row):
     return row[key]
 
 
+def _get_column_value_uncased(key: str, row: agate.Row) -> Any:
```

Comment:
this might be a better fit in core/dbt/clients/agate_helper.py

Reply:
Moved.